### PR TITLE
docs: support pipenv

### PR DESCRIPTION
**Pull request by robert-zaremba**
https://www.pivotaltracker.com/n/projects/2452320

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,14 +25,38 @@ The project specification is [published here](https://paper.dropbox.com/publishe
...
 
-+ algod v2.1.3+
++ algod v2.1.6-stable or higher
 
 Make sure that the node you are connecting to has a `"EnableDeveloperAPI": true` option set in the `<node_data>/config.json`. This is required to compile smart contracts using REST / SDK.
 
+
+### PyTeal
+
+`algob` supports TEAL smart contracts written in PyTeal. To use them, you have to have [pyteal](https://pypi.org/project/pyteal/) package available in your Python context:
```
Comment (ratik21) [inline README.md:40]:
```suggestion
`algob` supports TEAL smart contracts written in PyTeal. To use them, you need to have [pyteal](https://pypi.org/project/pyteal/) package available in your Python context:
```

File: packages/algob/README.md [hunk 2]
```diff
@@ -2,13 +2,17 @@
 
 This package provides the main entry point into the application.
 
+Please read the main [README](../../README.md) file for details about the project and `algob`. This file provides a development description about the `algob` package.
+
 ## Usage
-It can be included as a library by importing `internal/lib/lib.ts` or be run from command line.
+
+
+`albob` can be included as a library by importing `internal/lib/lib.ts` or be run from command line as described in the project [README](../../README.md) file.
```
Comment (ratik21) [inline packages/algob/README.md:10]:
```suggestion
`algob` can be included as a library by importing `internal/lib/lib.ts` or can be run from command line as described in the project [README](../../README.md) file.
```

File: README.md [hunk 3]
```diff
@@ -25,14 +25,38 @@ The project specification is [published here](https://paper.dropbox.com/publishe
...
+
+#### Using Pipenv
+
+We recommend to use [pipenv](https://pipenv.pypa.io) and use virtual environments. Pipenv is a packaging tool for Python that solves some common problems associated with the typical workflow using pip, virtualenv, and the good old requirements.txt. In addition to addressing some common issues, it consolidates and simplifies the development process to a single command line tool. It automatically creates and manages a virtualenv for your projects, as well as adds/removes packages from your Pipfile as you install/uninstall packages.
+
+With `pipenv` that you can use the `Pipfile` and `Pipfile.lock` files from this repository and copy it to your project. Then:
```
Comment (amityadav0) [inline README.md:46]:
```suggestion
With `pipenv` installed you can use the `Pipfile` and `Pipfile.lock` files from this repository and copy it to your project. Then:
```

File: README.md (hunk 1 above)
Comment (robert-zaremba) [reply README.md:40]: The original wording is better. https://www.quora.com/Is-have-to-have-correct-grammar-like-in-I-have-to-have-that

File: README.md (hunk 3 above)
Comment (robert-zaremba) [reply README.md:46]: thanks

**Review by amityadav0 — APPROVED** [review]

**Pull request merged**